app/src/main/java/de/westnordost/streetcomplete/osm/cycleway_separate/SeparateCyclewayItem.kt

```diff
@@ -20,7 +20,7 @@ private val SeparateCycleway.titleResId: Int get() = when (this) {
 
 private fun SeparateCycleway.getIconResId(isLeftHandTraffic: Boolean): Int = when (this) {
     PATH ->               R.drawable.ic_separate_cycleway_path
-    NOT_ALLOWED ->        R.drawable.ic_separate_cycleway_no
+    NOT_ALLOWED ->        R.drawable.ic_separate_cycleway_disallowed
```

Comment:
Strings that are not used anymore should be deleted from strings.xml

x2

Reply:
Ran a check, found and removed `R.string.separate_cycleway_allowed`, but not the second one.